src/commands/imageSource/buildImageInAzure/openImageErrorLog.ts

```diff
@@ -0,0 +1,41 @@
+import { sendRequestWithTimeout } from "@microsoft/vscode-azext-azureutils";
+import { nonNullValue } from "@microsoft/vscode-azext-utils";
+import { TextDocumentContentProvider, Uri, window, workspace } from "vscode";
+import { ext } from "../../../extensionVariables";
+import { IBuildImageInAzureContext } from "./IBuildImageInAzureContext";
+
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+const contentScheme: string = 'image-error-log';
```

Comment:
I feel that the scheme should be a bit more unique to our extension than this to avoid conflicting with any other providers.

Maybe `azurecontainerapps-imagelogs`?